components/config-mgmt-service/backend/elastic/nodes.go

```diff
@@ -83,7 +83,7 @@ func (es Backend) GetInventoryNodes(ctx context.Context, start time.Time,
 		Query(mainQuery).
 		Index(IndexNodeState).
 		Size(pageSize).
-		Sort(CheckinTimestamp, ascending).
+		Sort(sortField, ascending).
```

Comment:
⚠️ This is potentially a behavior change since we were previously ignoring this variable.  The linter was complaining about code on lines 70-72 being an ineffectual assignment.  We could instead continue to ignore this argument and then remove those lines.

Reply:
Whelp, a bunch of tests failed, so let me try ignoring this argument instead.